src/pubd/repository.rs

```diff
@@ -324,6 +326,8 @@ impl RepositoryContent {
     }
 
     pub fn session_reset(&mut self, config: &RepositoryRetentionConfig) -> KrillResult<()> {
+        info!("Perform RRDP session reset - ensures a consistent view for RPs in case we restarted from a backup.");
```

Comment:
Performing instead of perform? And perhaps also "rest - ensures" would be nicer as "reset. This ensures" ?

Reply:
done